database/sql/src/main/scala/cromwell/database/slick/MetadataSlickDatabase.scala

```diff
@@ -269,6 +262,40 @@ class MetadataSlickDatabase(originalDatabaseConfig: Config)
     runTransaction(action)
   }
 
+  private def summarizeRawMetadata(rawMetadataEntries: Seq[MetadataEntry],
```

Comment:
This looks like a lot of copy/paste from the `summarizeMetadata` function below. 

I _think_ you should be able to update `summarizeMetadata` to:

* Generate raw metadata in the same way as it does now, but then:
* Pass that set of raw metadata into this function to actually process it